### DISCUSSION TITLE
Suggest to people to submit info on their carts?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [sanni/cartreader#758] [Ideas & Suggestions]
I was thinking it might be good to suggest to people (on the github or maybe in the firmware itself) to submit info on their dumps to here or directly to No-Intro. That would help improve the quality of the data in the CRC lists in the firmware, plus just be good for game preservation in general.

**Comment by Ancyker** [comment]
As far as I'm aware, the CRC databases come from no-intro. So if something is wrong with them, then people would (and have) posted here about it. Though it is true that we don't know which games have been tested and which haven't.

**Reply by mariomadproductions** [reply]
Yeah I was partially thinking about when they *do* get a match - in those cases, they probably wouldn't get in touch, so the No-Intro entries won't get marked as verified.

(Also some people might not think to get in touch, even if there is a mismatch, as long as the ROM seems to work)

**Comment by nsx0r** [comment]
I'm with No-Intro and the OSCR is very useful, but they are 2 separate projects and I don't want to hijack this repo.
I'll just say this for the sake of preservation: any dump is always appreciated and everyone is welcome to submit them or ask for help, feel free to contact us on Discord or in the forums https://no-intro.org/

**Reply by mariomadproductions** [reply]
Yeah, I just think a lot of people might not know that even matching dumps are useful.
